components/datePicker/datePicker.module.scss

```diff
@@ -11,11 +11,17 @@
 
 
 .datePickerContainer {
-
+  display: flex;
+  flex-direction: column;
 
   height: 60px;
```

Comment:
Manual height setting is unnecessary because it adds an unexpected overflow to `.datePickerContainer` and shifts the offset between `DatePIckerContainer` and the child `Textearea` given by flex-gap on the parent container.